client/signup/jetpack-connect/new-site/wordpress-logo.jsx

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default () => (
```

Comment:
Any good reason why we want to keep this as a separate component instead of storing it in a separate .svg file?